arangod/Transaction/Manager.h

```diff
@@ -53,11 +53,14 @@ namespace transaction {
 class Context;
 struct Options;
 
-/// @bried Tracks TransasctionState instances
+/// @brief Tracks TransasctionState instances
 class Manager final {
-  static constexpr size_t numBuckets = 16;
-  static constexpr double defaultTTL = 10.0 * 60.0;   // 10 minutes
-  static constexpr double tombstoneTTL = 5.0 * 60.0;  // 5 minutes
+  static constexpr size_t numBuckets   = 16;
+  static constexpr double idleTTL   = 10.0;              // 10 seconds
+  static constexpr double totalTTL  = 60.0;              // 60 seconds
+  static constexpr double totalTTLDBServer  = 3 * 60.0;  // 6 minutes
```

Comment:
the code and the comment disagree here. If it's 6 minutes, then the code should be `6 * 60.0`. If 3 minutes, then the comment should be adjusted.